src/test/java/reposense/parser/CsvParserTest.java

```diff
@@ -139,6 +162,71 @@ public void merge_twoRepoConfigs_success() throws ParseException, IOException {
         Assert.assertEquals(REPO_LEVEL_GLOB_LIST, actualConfigs.get(0).getIgnoreGlobList());
     }
 
+    @Test
+    public void merge_emptyLocation_success() throws ParseException, IOException {
+        FIRST_AUTHOR.setIgnoreGlobList(FIRST_AUTHOR_GLOB_LIST);
+        SECOND_AUTHOR.setIgnoreGlobList(REPO_LEVEL_GLOB_LIST);
+
+        List<Author> expectedBetaAuthors = new ArrayList<>();
+        expectedBetaAuthors.add(FIRST_AUTHOR);
+        expectedBetaAuthors.add(SECOND_AUTHOR);
+
+        List<Author> expectedDeltaAuthors = new ArrayList<>();
+        expectedDeltaAuthors.add(FIRST_AUTHOR);
+
+        RepoConfiguration expectedBetaConfig =
+                new RepoConfiguration(new RepoLocation(TEST_REPO_BETA_LOCATION), TEST_REPO_BETA_BRANCH);
+        expectedBetaConfig.setAuthorList(expectedBetaAuthors);
+        expectedBetaConfig.setAuthorDisplayName(FIRST_AUTHOR, "Nbr");
+        expectedBetaConfig.setAuthorDisplayName(SECOND_AUTHOR, "Zac");
+        expectedBetaConfig.addAuthorAliases(SECOND_AUTHOR,  Arrays.asList("Zachary Tang"));
+        expectedBetaConfig.setIgnoreGlobList(REPO_LEVEL_GLOB_LIST);
+
+        RepoConfiguration expectedDeltaConfig =
+                new RepoConfiguration(new RepoLocation(TEST_REPO_DELTA_LOCATION), TEST_REPO_DELTA_BRANCH);
+        expectedDeltaConfig.setAuthorList(expectedDeltaAuthors);
+        expectedDeltaConfig.setAuthorDisplayName(FIRST_AUTHOR, "Nbr");
+
+        List<RepoConfiguration> expectedConfigs = new ArrayList<>();
+        expectedConfigs.add(expectedBetaConfig);
+        expectedConfigs.add(expectedDeltaConfig);
+
+        String input = String.format("-config %s", MERGE_EMPTY_LOCATION_FOLDER);
+        CliArguments cliArguments = ArgsParser.parse(translateCommandline(input));
+
+        List<RepoConfiguration> actualConfigs =
+                new RepoConfigCsvParser(((ConfigCliArguments) cliArguments).getRepoConfigFilePath()).parse();
+        List<RepoConfiguration> authorConfigs =
+                new AuthorConfigCsvParser(((ConfigCliArguments) cliArguments).getAuthorConfigFilePath()).parse();
+        RepoConfiguration.merge(actualConfigs, authorConfigs);
+
+        Assert.assertEquals(2, actualConfigs.size());
+        Assert.assertEquals(expectedConfigs, actualConfigs);
+
+        RepoConfiguration firstExpectedConfig = expectedConfigs.get(0);
+        RepoConfiguration secondExpectedConfig = expectedConfigs.get(1);
+        RepoConfiguration firstActualConfig = actualConfigs.get(0);
+        RepoConfiguration secondActualConfig = actualConfigs.get(1);
+
+        Assert.assertEquals(firstExpectedConfig.getLocation(), firstActualConfig.getLocation());
+        Assert.assertEquals(firstExpectedConfig.getAuthorList().hashCode(),
```

Comment:
You don't need to call `hashCode()` right?

```
Assert.assertEquals(firstExpectedConfig.getAuthorList(), firstActualConfig.getAuthorList());
```

Reply:
Yes, but the original `merge_twoRepoConfigs_success()` method also have this so I just followed, should I also remove this in that method?

Reply:
Yes. Please help me remove them. 👍